src/session.rs

```diff
@@ -7,7 +7,7 @@ use crypto::aes::KeySize;
 use crypto::aes_gcm::AesGcm;
 use crypto::chacha20poly1305::ChaCha20Poly1305;
 use crypto::scrypt::{scrypt, ScryptParams};
-use ring::rand::SystemRandom;
+use ring::rand::{SecureRandom, SystemRandom};
```

Comment:
Is this implication by adding this that `SystemRandom` is insecure? And would that mean you recommend changing the usage of `SystemRandom` to `SecureRandom` everywhere?